src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -4,9 +4,10 @@
 using System.IO;
 using System.Linq;
 using System.Net;
-using System.Reflection;
 using System.Security.Cryptography.X509Certificates;
 using System.Threading.Tasks;
+using static Azure.Functions.Cli.Common.OutputTheme;
+using static Colors.Net.StringStaticMethods;
```

Comment:
Curious, did you accidentally move this up?

Reply:
it was no in the right order. so fixed it.

Reply:
Well VS moved it up. :) when i selected sort